evm/src/NttManager/NttManager.sol

```diff
@@ -214,11 +216,6 @@ contract NttManager is INttManager, NttManagerState {
         );
     }
 
-    /// @inheritdoc INttManager
```

Comment:
I think we should keep this implementation around since this is called from the transceiver. We want the transceiver to fetch the immutable, not call the underlying token again